{{ cookiecutter.project_name }}/docker-compose.yml

```diff
@@ -25,20 +25,19 @@ services:
       REDIS_PASSWORD: "${REDIS_PASSWORD}"
     healthcheck:
       test: "redis-cli PING || exit 1"
-      interval: 1m
-      timeout: 10s
-      retries: 3
+      interval: 15s
+      timeout: 15s
+      retries: 4
 
-  webapp:
-    build: "./webapp"
+  migrate:
```

Comment:
IIUC, this "service" will run DB migrations? If so, does it need to be run every time? is there a need or a way to prevent it from being executed when running `docker-compose up`?

Reply:
Or is running DB migration every time compose runs on purpose?